CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Improvements
 * Now uses [OpenSSL 1.1.1f](https://www.openssl.org/source/openssl-1.1.1f.tar.gz). [PR #97](https://github.com/corretto/amazon-corretto-crypto-provider/pull/97)
+* **EXPERIMENTAL** support for aarch64 added. [PR #99](https://github.com/corretto/amazon-corretto-crypto-provider/pull/99)
```

Comment:
I would suggest moving to something like https://www.conventionalcommits.org/en/v1.0.0/ to handle this.
Out of scope for this PR, just a note.

Reply:
We are not changing our versioning of ACCP at this time.